config/e2e/batch_job.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       containers:
         - name: e2e
           image: {{ .E2EImage }}
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
I remember this conversation: https://github.com/elastic/cloud-on-k8s/pull/1995#discussion_r335046933.
Is it important to cache the image?

Reply:
Without this change tests will not work in `Kind`. For unknown reasons for me, it can't pull image. We even have a special script to load images into `kind` https://github.com/elastic/cloud-on-k8s/blob/master/hack/kind/kind.sh#L137
Probably @barkbay can explain better why do we need to load images in `kind` when he will be back...

Reply:
IIRC, the docker images need to be pre-loaded into Kind nodes because there's no registry to pull the images from. When I changed the image pull policy, we didn't have Kind tests yet and it was just convenient to have that setting committed to Git to help test uncommited changes in a GKE cluster. 

I think reverting to `IfNotPresent` makes sense now.